src/Map/src/Renderer/AbstractRenderer.php

```diff
@@ -81,4 +81,21 @@ final public function renderMap(Map $map, array $attributes = []): string
 
         return \sprintf('<div %s></div>', $stimulusAttributes);
     }
+
+    private function getMapAttributes(Map $map): array
+    {
+        $computeId = fn (array $array) => hash('xxh3', json_encode($array));
+
+        $attrs = $map->toArray();
+
+        foreach ($attrs['markers'] as $key => $marker) {
+            $attrs['markers'][$key]['@id'] = $computeId($marker);
+        }
+
+        foreach ($attrs['polygons'] as $key => $polygon) {
+            $attrs['polygons'][$key]['@id'] = $computeId($polygon);
+        }
```

Comment:
If the idea is to have reproductible identifiers, i think some more normalization would be required, no ?

Reply:
That's possible! I think you mean about attributes ordering like it's done in LiveComponent hydration/checksum processes?

Well, since we (de)normalize with fromArray and toArray, the array structure is identical, so I believe it is not necessary...?

Reply:
Ok! Just curious about it... and if needed as you said we now how to implement it :)